README.md

```diff
@@ -38,5 +38,81 @@ rails generate blacklight:install
 * Bundler
 * Rails 5.0+
 
-## Configuring Apache Solr 
+## Configuring Apache Solr
 You'll also want some information about how Blacklight expects [Apache Solr](http://lucene.apache.org/solr ) to run, which you can find in [README_SOLR](https://github.com/projectblacklight/blacklight/wiki/README_SOLR)
+
+## Building the javascript
+The javascript is built by npm from sources in `app/javascript` into a bundle
+in `app/assets/javascripts/blacklight/blacklight.js`. This file should not be edited
+by hand as any changes would be overwritten.
+
+This is accomplished with the following steps:
+1. Install npm
+1. run `npm install` to download dependencies
+1. run `npm run js-compile-bundle` to build the bundle
+1. run `npm publish` to push the javascript package to https://npmjs.org/package/blacklight-frontend
+
+## Using the javascript
+Install Webpacker
```

Comment:
Either add a link to [Webpack's install page](https://webpack.js.org/guides/installation/) or include :
```
npm install --save-dev webpack
```

Reply:
Webpacker rather than Webpack: https://github.com/rails/webpacker#installation